CMakeLists.txt

```diff
@@ -242,11 +244,11 @@ if(VW_INSTALL)
     SET (BOOST_LINK_LIBRARIES "${BOOST_LINK_LIBRARIES} ${BOOST_LIBRARY}")
   ENDFOREACH()
 
-  configure_file(libvw.pc.in libvw.pc)
-  configure_file(libvw_c_wrapper.pc.in libvw_c_wrapper.pc)
+  configure_file(libvw.pc.in libvw.pc @ONLY)
```

Comment:
What does `@ONLY` do here?

Reply:
@ONLY means CMake will only expand variables in the form @var@ and not ${var}. This is required becuase pkgconfig file use vars of the form ${var} and we don't want them expanded at configure time.

Reply:
Makes sense. Thanks for the explanation :)